### ISSUE TITLE
SIGMOD'22 Submission

### ISSUE THREAD

**Issue by geoffxy** [mitdbg/treeline#15]
## Paper writing (first draft, section checklist)
- [x] Abstract
  - Andreas
- [x] Introduction
  - Andreas
- [x] Design
  - Markos
- [x] Evaluation
  - Geoffrey (+ experiments)
- [x] Background
- [x] Related Work
- [x] Discussion
- [x] Conclusion

## Experiments Must Have
- [x] Lean store bindings for comparison
- [x] YCSB synthetic
- [x] YCSB Bourbon datasets
  - NOTE: These YCSB experiments need to be re-run after #12 is addressed
- [x] Test experiments with LeanStore

## Experiments nice to have
- [ ] Kvell bindings finished and comparison
- [ ] SOSD bindings for LeanStore too so we can compare against LLSM
- [ ] SOSD benchmark run for comparison (LLSM / RocksDB)
- [x] #18 

## Required code changes
- [x] #12
- [x] #10
- [x] Adapt experiments to LeanStore (cannot "reopen" a database)

## Features (nice to have)
- ALEX page table improvements
  - [ ] Optimistic lock coupling to remove model-wide mutex
  - [ ] Compression for adjacent physical pages
